README.md

```diff
@@ -18,3 +18,25 @@ $ xe host-call-plugin host-uuid=${HOST_UNDER_TEST_UUID} plugin=updater.py fn=che
 ```
 $ xe host-call-plugin host-uuid=${HOST_UNDER_TEST_UUID} plugin=updater.py fn=update
 ```
+
+## Proxy configuration
+
+### `get_global_proxy()`:
```

Comment:
Is it a proxy for all connections or only for `yum`?

Reply:
It's only for yum.

Reply:
Maybe it should be more explicit in the function name then.